### ISSUE TITLE
New rule: builtin `input()` should not be called from async function

### ISSUE THREAD

**Issue by Zac-HD** [python-trio/flake8-async#199]
```python
async def fn():
    ans = input("what to say")  # bad - no task can run while waiting for the user
    ans = await trio.to_thread.run_sync(input, "what to say")  # good, if inelegant
```

We could add this to TRIO222, or give it a new error code specific to this function.

**Comment by jakkdl** [comment]
I'll re-bump [adding pre-commit.ci](https://github.com/Zac-HD/flake8-trio/issues/187#issuecomment-1568021897), will probably need to do a minor pass on any new changes in testing tools etc so I don't have to lump those into a feature PR.

**Comment by jakkdl** [comment]
note that this could be achieved with use of ASYNC200 and `async200-blocking-calls`, but agree it makes sense as a default rule.
It doesn't quite fit with any 2xx rules, so I'm creating a new one for it.
